extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/util/AuthZUtils.scala

```diff
@@ -175,4 +190,38 @@ private[authz] object AuthZUtils {
   def quote(parts: Seq[String]): String = {
     parts.map(quoteIfNeeded).mkString(".")
   }
+
+  private def verifyKyuubiSessionUser(spark: SparkContext, user: String): Unit = {
+    val isSessionUserVerifyEnabled =
+      spark.getConf.getBoolean(s"spark.kyuubi.session.user.sign.enabled", defaultValue = false)
+    if (isSessionUserVerifyEnabled) {
+      val isVerified = {
+        try {
+          val userPubKeyStr = spark.getLocalProperty("kyuubi.session.sign.publickey")
+          val userSign = spark.getLocalProperty("kyuubi.session.user.sign")
+          verifySignWithECDSA(user, userSign, userPubKeyStr)
```

Comment:
`userSign` and `userPubKeyStr` may be null

Reply:
Yes, let's check them not null.